android/fastlane/metadata/android/en-GB/changelogs/10032.txt

```diff
@@ -0,0 +1,13 @@
+Added
```

Comment:
+ option to set a custom icon for a website (https://github.com/student-hub/acs-upb-mobile/pull/233)